tachyons-workshop/tachyons-postcss-setup/postcss.config.js

```diff
@@ -1,9 +1,12 @@
 module.exports = {
+  /* each of these plugins allows you to write css with special syntax
+     Or transforms the output into a more desirable format
+  */
   plugins: [
-    require('postcss-import'),
-    require('autoprefixer'),
-    require('postcss-custom-media'),
-    require('postcss-clean'),
-    require('postcss-custom-properties')
+    require('postcss-import'), // allows you to use @import
+    require('autoprefixer'), // adds vendor prefixes like -webkit or -moz to your css for better support
+    require('postcss-custom-media'), // can set media queries as variables
+    require('postcss-clean'), // minifies the output css (i.e. removes all the spaces and comments)
```

Comment:
I had an issue before where if I didn't have `postcss-clean` last it would error. Not sure if that could be an issue here?